.prow/scripts/test-serving.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+.prow/scripts/download-maven-cache.sh \
+    --archive-uri gs://feast-templocation-kf-feast/.m2.2019-10-24.tar \
+    --output-dir /root/
+
+# Skip Maven enforcer: https://stackoverflow.com/questions/50647223/maven-enforcer-issue-when-running-from-reactor-level
```

Comment:
@ches Do you have any thoughts on this? I'm not well versed enough with Maven or the enforcer plugin to comment.

Reply:
Hmm, I hadn't encountered this but I can reproduce it with `mvn --projects serving test`.

However a solution rather than skipping Enforcer seems to be to include `--also-make`, that works for me (short version for day-to-day typing: `-am`).

That isn't normally strictly necessary since `serving` doesn't have any inter-module dependency (like `core` does on `ingestion`), but it seems like specifying it makes it run as a Reactor build that includes `serving` and the parent POM together, so that their relationship can be verified.

`--also-make` is often what you want in multi-module projects. @smadarasmi had a related question about running components after the dependency management changes that prompted me to make additions to development notes, I should have a PR with those today.

Reply:
Also, I'm not sure the `install` step is really needed here FWIW, just `test` should suffice for unit tests.

Reply:
I've tried making the change to `--also-make` in this Prow script in #276. Not sure if the CI build is functional yet, but maybe I can learn how to try it.

Reply:
Yeah we can try those options.
I added the install script before test to follow what Travis CI does for Maven project
https://docs.travis-ci.com/user/languages/java/#maven-dependency-management
But I guess we can also omit it also because it makes it simpler.

Currently the test worflow is derived from `config.yaml` in `.prow` folder. Updating config.yaml is not that straightforward because the `config_updater` plugin in Prow only updates the `config.yaml` at a certain interval after merge to master. So I currently update the config.yaml manually. 

But if you update say `test-serving.sh`, the CI should use the updated `test-serving.sh`